lib/Mail/DMARC/Report/Sender.pm

```diff
@@ -463,7 +463,7 @@ sub log_output {
 
         my @parts;
         foreach my $key ( sort keys %$args ) {
-            my $value = $args->{ $key };
+            my $value = $args->{ $key } // '';
```

Comment:
this solves the error, but you're going to get output such as `key1=val, key2=`.  If there's value in logging an empty key, okay. If not, skipping to the next iteration of the for loop might be better.

Reply:
I would still like to log it, there key may still be relevant